cluster/addons/registry/registry-pv.yaml.in

```diff
@@ -0,0 +1,16 @@
+kind: PersistentVolume
+apiVersion: v1
+metadata:
+  name: kube-system-kube-registry-pv
+  labels:
+    kubernetes.io/cluster-service: "true"
+spec:
+{% if pillar.get('cluster_registry_disk_type', '') == 'gce' %}
```

Comment:
So if cluster_registry_disk_type isn't `gce`, this yaml blob just trails off at `spec:`?